.github/workflows/pr.yml

```diff
@@ -1,7 +1,7 @@
 name: Check PR
 
 on:
-  pull_request_target:
```

Comment:
https://woodruffw.github.io/zizmor/audits/#dangerous-triggers

Reply:
https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#event-triggers

Reply:
I'm in favor of removing of this check with something else, but not right now.